apps/urls.py

```diff
@@ -46,6 +47,8 @@
         yasg_schema_view.with_ui('swagger', cache_timeout=None), name='schema_swagger'),
     url(r'^api/docs-redoc/$',
         yasg_schema_view.with_ui('redoc', cache_timeout=None), name='schema_redoc'),
-    
-    url(r'^site_test/', site_test.index, name='site_test')
+    url(r'^site_test/', site_test.index, name='site_test'),
+    # This is the challenge from cerbot (certbot.eff.org) after running "sudo certbot certonly --manual"
+    url(r'^\.well-known/acme-challenge/(?P<acme>.+)',
+        lambda request, acme: HttpResponse('%s.fP2MtOMJg03pQ5U9zfjwPdFzA-12z143KjztlvCkMqc' % acme))
```

Comment:
Could we open an issue(s) concerning what remains to be done to implement HTTPS?

Reply:
TL DR: we need to have service provider supporting https and it should just work.

This line is not directly related to implementing https as it is just challenge to prove ownership of domain for the third party issuing certificate. After obtaining certificate, it is passed to infrastructure service (Google APP or Heroku) and used by front-line server that handles https and futher on through http to our minimalistic python http app.